src/IO/H5/VolumeData.cpp

```diff
@@ -266,6 +360,10 @@ void VolumeData::write_volume_data(
   // Write the Connectivity
   h5::write_data(observation_group.id(), total_connectivity,
                  {total_connectivity.size()}, "connectivity");
+  if (not pole_connectivity.empty()) {
```

Comment:
why does the pole connectivity need to be separate? Can't it just be part of the connectivity? If not, it should be commented why it can't

Reply:
The pole connectivity is separate because it uses triangles, not quads, and I couldn't figure out how to tell the visualization script to draw quads for some connectivities but triangles for others. I'll add a comment along these lines

Reply:
Ah right, I remember talking about this :)